packages/button/src/Button.js

```diff
@@ -264,11 +263,23 @@ const Root = createStyledComponent('button', buttonStyles, {
 });
 const Content = createStyledComponent('span', contentStyles);
 
+const Inner = createStyledComponent('span', (props, theme) => ({
+  alignItems: 'center',
+  display: 'inline-flex',
+  justifyContent: 'center',
+  width: '100%',
+  '& > svg': {
```

Comment:
We should probably target this as `[role=icon]` rather than `svg`.  This way, it at least has a chance of working with other non-svg icon components.